includes/fields/class-field-post-taxonomy.php

```diff
@@ -215,25 +215,44 @@ public function tax_ajax( $post_id = null ) {
     }
 
     public function tax_select( $post_id = null ) {
-        $attr = $this->field_settings;
-
+        $attr     = $this->field_settings;
         $selected = $this->terms ? $this->terms[0] : '';
-
         $required = sprintf( 'data-required="%s" data-type="select"', $attr['required'] );
 
+        $parents      = $this->field_settings['exclude'];
```

Comment:
@hasinur1997 You've already set `$attr     = $this->field_settings;`, so use $attr in the rest of the method whenever you need `$this->field_settings`